entities/card.py

```diff
@@ -1,32 +1,44 @@
 from dataclasses import dataclass
-from entities.card_type import CardType
+
+from flask import Request
+
 from entities.card_design_properties import CardDesignProperties
-from datetime import datetime
-from uuid import UUID
-from typing import Dict
-from card_type import CardType
+from typing import Dict, List
+from entities.card_proprties import CardProperties
+from consts import *
 
+CARDS_SERIALIZED = Dict[str, List[Dict[str, str]]]
 
 
 @dataclass
 class Card:
-    content: str
-    type: CardType
+    card_properties: CardProperties
```

Comment:
Are you sure this is a good idea to put all the card's properties in a different class? 
I think it make the card class meaningless if it's not the one holding it's properties.